src/Cake.Bridge/Cake.Bridge.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netstandard2.0;net46</TargetFrameworks>
+    <TargetFrameworks>net5.0;netstandard2.0;net461</TargetFrameworks>
```

Comment:
`Cake.Core` / `Common` still targets `net46` so see no need to potentially break things by changing this.

Reply:
Reverted && push --force